test/ad/utils.jl

```diff
@@ -47,7 +47,7 @@ function test_ad(f, x, broken=(); rtol=1e-6, atol=1e-6)
         end
     end
 
-    if AD == "All" || AD == "Enzyme"
+    if (AD == "All" || AD == "Enzyme") && VERSION >= v"1.10"
         if :EnzymeReverse in broken
```

Comment:
The Julia compiler bug on 1.6 only applies to [batched] fwd mode. It shouldn't be hit for reverse mode if you want to losen the tests

Reply:
Nice, changed that. Is `forward_broken = :EnzymeForward in broken || :Enzyme in broken || VERSION <= v"1.6"` correct, or did the compiler bug get fixed only later?